pkg/ownerreferences/ownerreferences.go

```diff
@@ -0,0 +1,74 @@
+package ownerreferences
```

Comment:
missing copyright boilerplate (should be the same of the controller-runtime package)